### DISCUSSION TITLE
Is there a way to have a column view with file size/date/etc. like in ls -l and every file manager?

### DISCUSSION THREAD

**Discussion by yanshay** [sxyazi/yazi#268] [1. Q&A]
Is there a way to see a column view with file size/date/etc. like in `ls -l`?

It seems like a basic requirement for going through files in any file manager but I can't find how to configure yazi to show it.
If it's not available, is there any decision not to include that or just didn't get to that yet?

**Comment by sxyazi** [comment]
Dup of https://github.com/sxyazi/yazi/issues/220

Will be shipped with Yazi v0.1.5 soon.
